index.html

```diff
@@ -431,12 +449,71 @@
 application caches</a> or from local resources.</p>
 </dd>
 
+<dt>readonly attribute unsigned short transferSize</dt>

```

Comment:
They get inherited so unless the text description differs, we could simply drop them.